src/LoRa_APRS_Tracker.cpp

```diff
@@ -280,25 +308,61 @@ void loop() {
     } else {
       speed_zero_sent = 0;
     }
-
+    telemetry();
+    String TEMPERATURE = String(TEMP);
+    String PRESSURE    = String(PRESS);
+    // REMOVE LAST CHARACTER FROM PRESSURE
+    PRESSURE.remove(PRESSURE.length() - 1); // PRESSURE ON APRS IS ONLY 5 DIGITS
+
+    // Uncomment to enable APRS telemetry for sensors with more data.
+
+    /// String HUMIDITY = String(HUMID);
+    /// String WIND_SPEED    = String(WIND_S);
+    /// String WIND_DIRECTION    = String(WIND_DIR);
+    /// String RAINX = String(RAIN);
+
+    // Using ... notifies APRS that the fields are NULL thus not showing up
+    // in the telemetry.
+    String HUMIDITY       = "...";
+    String WIND_SPEED     = "...";
+    String WIND_DIRECTION = "...";
+    String RAINX          = "...";
     String aprsmsg;
-    aprsmsg = "!" + lat + DBEACON_OVERLAY + lng + DBEACON_SYMBOL + course_and_speed + alt;
-    // message_text every 10's packet (i.e. if we have beacon rate 1min at high
-    // speed -> every 10min). May be enforced above (at expirey of smart beacon
-    // rate (i.e. every 30min), or every third packet on static rate (i.e.
-    // static rate 10 -> every third packet)
-    if (!(rate_limit_message_text++ % 10)) {
-      aprsmsg += DBEACON_MESSAGE;
+    if (WEATHER_DATA == true) {
+
+      Serial.print("Weather data enabled");
+      aprsmsg = "!" + lat + DBEACON_OVERLAY + lng + DBEACON_SYMBOL + WIND_DIRECTION + "/" + WIND_SPEED + "g" + RAINX + "t" + TEMPERATURE + "h" + HUMIDITY + "b" + PRESSURE;
```

Comment:
Maybe send out Weather as real APRS Telemetry messages in a future Version.

Reply:
Can you elaborate what you mean real APRS Telemetry ? In this case it will show up on APRS.FI which only the values specified. When sending out "..." it will just ignore them, hence why I left them there so if you connect some more sophisticated sensor you could uncomment the top portion.

Reply:
APRS has a Spec for Telemetry packets, not just adding text to packet comment.
This draws nice graphs on aprs.fi in the Telemetry section. 

I have done so in my solar powered digipeater, see 
https://github.com/dj1an/LoRa-APRS-Digipeater/blob/main/src/main.cpp#L401

Would be nice2have in future Versions, but for now we could leave it as it is

Reply:
For Weather Data there's the specific packet format that covers all this data.
Starting on Page 62: http://www.aprs.org/doc/APRS101.PDF

aprs.fi renders nice graphs specific to the weather and generates summary reports based on local stations.
https://aprs.fi/weather/a/EW8865